opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -172,6 +172,7 @@ where
         let mut log_record = self.logger.create_log_record();
         log_record.set_severity_number(severity_of_level(meta.level()));
         log_record.set_severity_text(meta.level().to_string().into());
+        log_record.set_target(meta.target().to_string());
```

Comment:
the logger created by appender  - please modify it to not populate version, as it'll be misleading now.

Reply:
also - there is some experimental feature where target was added as an attribute. lets remove that now?

Reply:
> the logger created by appender - please modify it to not populate version, as it'll be misleading now.

Other thought  - we can keep it, and allow users to customize the logger-name, for those interested to have the single logger, and use that created by appender?

> also - there is some experimental feature where target was added as an attribute. lets remove that now?

Yeah good point. This is basically the "log.target" coming from log-to-tracing conversion. Will handle it as separate PR ?

Reply:
the version will be misleading. it'll be the version of appender-crate, but the name will be that of the tracing:target. I'd suggest to remove version completely to avoid this confusion..

Reply:
Agree, as of now I have removed `version` and `attributes` population from OTLP exporter. We should make this configurable in OTLP exporter - default to override the scope.name with target, unless configured otherwise. Will create a separate PR for that (and a tracing task). I think the current way of configuration is bit messy with no separate way to provide log/trace/metrics specific configuration.

Reply:
#1890 to track this.

Reply:
> This is basically the "log.target" coming from log-to-tracing conversion. Will handle it as separate PR ?
Lets add an issue to track this too.

Reply:
#1894 for tracking.